src/components/subgoal/Subgoal.tsx

```diff
@@ -23,28 +60,95 @@ const Subgoals = ({ subgoal }: SubgoalProps) => {
           padding: "1rem",
         }}
       >
-        <p>{subgoal.description}</p>
+        <Typography
+          sx={{ color: "var(--primary-40)" }}
+          variant="overline"
+          display="block"
+          gutterBottom
+        >
+          #{(index ?? 0) + 1} created on {format(subgoal.created_at, "P")}
+        </Typography>
+        <Box sx={{ display: "flex", justifyContent: "space-between" }}>
+          <Box sx={{ display: "flex" }}>
+            <ContentPasteIcon
+              sx={{
+                color: "var(--grey-10)",
+                fontSize: 12,
+                margin: "1.25rem",
+                marginLeft: "0.5rem",
+                marginRight: "0.5rem",
+              }}
+            />
+
+            <Box sx={{ margin: "1rem", marginLeft: ".5rem" }}>
+              {subgoal.description}
+            </Box>
+          </Box>
+          <Box sx={{ margin: "1rem" }}>Edit</Box>
```

Comment:
Link or Tertiary button with placeholder logic

Reply:
This button is now a link which gives a placeholder alert when clicked.